.expeditor/config.yml

```diff
@@ -0,0 +1,33 @@
+# Documentation available at http://expeditor-docs.es.chef.io/
+
+product_key: marketplace
+
+slack:
+  notify_channel: supermarket-notify
+
+github:
+  version_file: VERSION
+  changelog_file: CHANGELOG.md
+  maintainer_group: chef-partners/partner-engineering
+
+merge_actions:
+  - built_in:bump_version:
+      ignore_labels:
+        - "Version: Skip Bump"
+        - "Expeditor: Skip All"
+  - built_in:update_changelog:
+      ignore_labels:
+        - "Changelog: Skip Update"
+        - "Expeditor: Skip All"
+  - built_in:trigger_omnibus_build:
+      ignore_labels:
+        - "Omnibus: Skip Build"
+        - "Expeditor: Skip All"
+
+artifact_actions:
+  promoted_to_unstable:
+  promoted_to_current:
+  promoted_to_stable:
+    - built_in:rollover_changelog
+    - built_in:create_github_release
```

Comment:
I don't think you need to do this. Releases for tags are automatic.

Reply:
Unless you're looking for releases like this: https://github.com/chef/inspec/releases